step-framework-timeseries/src/main/java/step/core/timeseries/TimeSeriesCollection.java

```diff
@@ -42,13 +43,16 @@ public TimeSeriesCollection(Collection<Bucket> collection, TimeSeriesCollectionS
         if (settings.getResolution() <= 0) {
             throw new IllegalArgumentException("The resolution parameter must be greater than zero");
         }
+        validateTtl(settings.getTtl());
         this.collection = collection;
         this.resolution = settings.getResolution();
         this.ttl = settings.getTtl();
         TimeSeriesIngestionPipelineSettings ingestionSettings = new TimeSeriesIngestionPipelineSettings()
                 .setResolution(settings.getResolution())
-                .setFlushingPeriodMs(settings.getIngestionFlushingPeriodMs());
+                .setFlushingPeriodMs(settings.getIngestionFlushingPeriodMs())
+                .setHandledAttributes(settings.getIgnoredAttributes());
```

Comment:
we're using two contradictory terms here, I guess setHandledAttributes should be changed for setIgnoredAttributes